.github/workflows/bindings_python.yml

```diff
@@ -82,6 +86,10 @@ jobs:
       - uses: actions/checkout@v4
       - name: Setup Rust toolchain
         uses: ./.github/actions/setup
+      - name: Ruff Check
+        uses: jpetrucciani/ruff-check@0.1.0
```

Comment:
Install ruff should be simple by calling pip install. Please don't use extra actions so that we don't need to review and maintain them.